packages/twenty-front/src/pages/settings/serverless-functions/SettingsServerlessFunctionDetail.tsx

```diff
@@ -70,13 +78,56 @@ export const SettingsServerlessFunctionDetail = () => {
     };
   };
 
-  const handleExecute = async () => {
-    await handleSave();
+  const resetDisabled =
+    !isDefined(latestVersionCode) || latestVersionCode === formValues.code;
+  const publishDisabled = latestVersionCode === formValues.code;
```

Comment:
we should prevent invalid function from being published too. Either in this PR either adding a TODO?

Reply:
done